src/util/ConstexprUtils.h

```diff
@@ -63,8 +63,8 @@ void ConstExprForLoop(const std::index_sequence<ForLoopIndexes...>&,
 template <size_t MaxValue, typename Function>
 void RuntimeValueToCompileTimeValue(const size_t& value,
                                     const Function& function) {
-  AD_CONTRACT_CHECK(value <= MaxValue); // Is the value valid?
-  ConstExprForLoop(std::make_index_sequence<MaxValue>{},
+  AD_CONTRACT_CHECK(value <= MaxValue);  // Is the value valid?
+  ConstExprForLoop(std::make_index_sequence<MaxValue + 1>{},
```

Comment:
Should be called `ConstexprForLoop` (small `e`).

Reply:
Done, see [57bf177](https://github.com/ad-freiburg/qlever/pull/860/commits/57bf17777fd4894254e6cdcf65b40eba1c745430).